### PR TITLE
Django REST Framework

### PR DESCRIPTION
**Pull request by gouzouni625**

### CODE REVIEW

**Comment by gouzouni625** [comment]
Please review @themiszamani @makistsantekidis .

File: webapp/api-doc/docs/LambdaInstanceDestroy.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ will also return a "202 ACCEPTED" code.
 
 ### Headers
 
-Type | Description | Required | Default value | Example value 
+Type | Description | Required | Default value | Example value
 ------|-------------|----------|---------------|---------------
-x-api-key | ~okeanos authentication token. If you have an account you may find the authentication token at (Dashboad-> API Access) https://accounts.okeanos.grnet.gr/ui/api_access. | `Yes` | None | tJ3b3f32f23ceuqdoS_..
+Authorization | ~okeanos authentication token. If you have an account you may find the authentication token at (Dashboad-> API Access) https://accounts.okeanos.grnet.gr/ui/api_access. | `Yes` | None | tJ3b3f32f23ceuqdoS_..
```
Comment (pkoro) [inline webapp/api-doc/docs/LambdaInstanceDestroy.md]: Last column should be `Token tJ3b3f32f23ceuqdoS_..` I think. Fix also curl example below.

File: webapp/backend/views.py [hunk 2]
```diff
@@ -208,202 +93,166 @@ def delete(self, request, format=None):
...
-    if not auth_url:
+        tasks.lambda_instance_stop.delay(data['uuid'], auth_url, auth_token, master_id,
+                                          slave_ids)
+
+        # Create event to update the database.
+        events.set_lambda_instance_status.delay(data['uuid'], LambdaInstance.STOPPING)
+
+        return Response({"result": "Accepted"}, status=status.HTTP_202_ACCEPTED)
+
+    def destroy(self, request, pk, format=None):
```
Comment (pkoro) [inline webapp/backend/views.py]: missing verb here I think

**Comment by pkoro** [comment]
@gouzouni625 check all the md files for similar mistakes like the one I commented above. Also check all functions in `views` for http verbs.

**Comment by pkoro** [comment]
+1

**Comment by makistsantekidis** [comment]
+1

**Pull request merged**